python-3.13.yaml

```diff
@@ -10,8 +10,8 @@ package:
     memory: 8Gi
   dependencies:
     provides:
-      - python3=0
-      - python-3=0
+      - python3=${{package.full-version}}
+      - python-3=${{package.full-version}}
     runtime:
```

Comment:
this and the other -dev one will make it the default. Are we ready? has everything completed rebuilding for 3.13?

Reply:
I doing think we should do this yet.